internal/backends/postgresql/query.go

```diff
@@ -36,27 +36,35 @@ import (
 // For capped collection with onlyRecordIDs, it returns select clause for recordID column.
 //
 // For capped collection, it returns select clause for recordID column and default column.
-func prepareSelectClause(schema, table string, capped, onlyRecordIDs bool) string {
+func prepareSelectClause(schema, table, comment string, capped, onlyRecordIDs bool) string {
+	if comment != "" {
+		comment = strings.ReplaceAll(comment, "/*", "/ *")
+		comment = strings.ReplaceAll(comment, "*/", "* /")
+		comment = `/* ` + comment + ` */` + ` `
+	}
+
 	if capped && onlyRecordIDs {
 		return fmt.Sprintf(
-			`SELECT %s FROM %s`,
+			`SELECT %s%s FROM %s`,
```

Comment:
@noisersup I had kept this together as to avoid unnecessary spacing. If the comment string is empty, the resulting query would have double space after the `SELECT` keyword. Of course this should not cause an execution error.